changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.5.1 | t.b.d.
+
+#### Bug fixes
+
+* Fixed brittle test which was testing tracking a `Kymograph` generated from an artificial `ImageStack`. This test erroneously attempted tracking a line with the threshold set to the minimum pixel intensity of the image. A threshold set to the minimum intensity is typically rejected by `track_greedy`, but in the case of this test, floating point errors ended up making the minimum pixel value end up slightly lower or higher than the threshold. The former was more common making the test fail only sporadically.
```

Comment:
I don't think we usually put this kind of internal stuff in the changelog, do we? At any rate, is it really a bug fix?

Reply:
Yeah, you are right. When I initially opened this PR, I thought we may still have to do a hotfix release to get it to pass on `conda-forge`, in which case a re-release without any notes would have been pretty strange.

Rerunning the action ended up making the error disappear though (different runner/env maybe), so I'll drop the entry.